images/node/Dockerfile

```diff
@@ -10,13 +10,6 @@
 #
 FROM openshift/origin
 
-COPY centos-paas-sig-openshift-origin36.repo /etc/yum.repos.d/
-COPY usr/bin/* /usr/bin/
-COPY opt/cni/bin/* /opt/cni/bin/
-COPY etc/cni/net.d/* /etc/cni/net.d/
-COPY conf/openshift-sdn-ovs.conf /usr/lib/systemd/system/origin-node.service.d/
-COPY scripts/* /usr/local/bin/
-COPY system-container/system-container-wrapper.sh /usr/local/bin/
 COPY system-container/manifest.json system-container/config.json.template system-container/service.template system-container/tmpfiles.template /exports/
```

Comment:
@smarterclayton @giuseppe let's determine for certain whether we do or do not want these files in the RPM. Also, what the generally accepted manner is to support system containers. I know @tdawson has been asked to set up new system container image creations in Brew, right? If we're doing those builds separately, do we need these files in the Origin container images?